torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -316,24 +316,22 @@ def analyze_kernel_mutations(functions, fn_name, num_args):
 
         if op.name == "tt.call":
             assert op.fn_call_name in functions
-            mutations = analyze_kernel_mutations(
+            if mutations := analyze_kernel_mutations(
```

Comment:
No need for a wallrus here, `extend()` can be called on an empty iterator (it will just be a noop). Feel free to drop the list comprehensions.